backend/src/seeds/factory/creation.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
+import { createPendingCreation, confirmPendingCreation } from '@/seeds/graphql/mutations'
+import { login } from '@/seeds/graphql/queries'
+import { CreationInterface } from '@/seeds/creation/CreationInterface'
+import { ApolloServerTestClient } from 'apollo-server-testing'
+import { User } from '@entity/User'
+import { AdminPendingCreation } from '@entity/AdminPendingCreation'
+// import CONFIG from '@/config/index'
+
+export const creationFactory = async (
+  client: ApolloServerTestClient,
+  creation: CreationInterface,
+): Promise<void> => {
+  const { mutate, query } = client
+
+  // get Peter Lustig
+  const peterLustig = await User.findOneOrFail({ where: { email: 'peter@lustig.de' } })
+  const variables = { ...creation, moderator: peterLustig.id }
```

Comment:
consider to move this block below the login and adjust the comment:

```suggestion
  // get Peter Lustig
  const peterLustig = await User.findOneOrFail({ where: { email: 'peter@lustig.de' } })
  const variables = { ...creation, moderator: peterLustig.id }
```
```suggestion
  // get Peter Lustig's user id
  const peterLustig = await User.findOneOrFail({ where: { email: 'peter@lustig.de' } })
  const variables = { ...creation, moderator: peterLustig.id }
```